website/project/views/node.py

```diff
@@ -718,6 +723,9 @@ def _view_project(node, auth, primary=False):
     anonymous = has_anonymous_link(node, auth)
     widgets, configs, js, css = _render_addon(node)
     redirect_url = node.url + '?view_only=None'
+    contributor_ids = []
+    for contributor in node.contributors:
+        contributor_ids.append(contributor._id)
```

Comment:
> No need to change it on this PR

Actually, looking at the code in context, I think we should switch to an list comprehension inline at (https://github.com/CenterForOpenScience/osf.io/pull/4932#discussion_r51457390) to conform to the existing style here.